week-4/C-more-complex-objects/exercise-3.js

```diff
@@ -32,23 +32,43 @@ var parkAvenueHouse = {
 
 // returns the full name (first name + last name) of the owner of the house
 function getOwnerFullName(house) {
-
+  return (
+    kinningParkHouse.currentOwner.firstName +
+    " " +
+    kinningParkHouse.currentOwner.lastName
+  );
 }
 
 // returns an array of the owners' email addresses of the two houses
 function getEmailAddresses(house1, house2) {
-
+  return (
+    kinningParkHouse.currentOwner.email +
+    ", " +
+    parkAvenueHouse.currentOwner.email
+  );
 }
 
 // returns the address for the cheapest house out of the two
 function getCheapestAddress(house1, house2) {
-
+  if (kinningParkHouse.price < parkAvenueHouse.price) {
+    return kinningParkHouse.address;
+  } else {
+    return parkAvenueHouse.address;
+  }

```

Comment:
good logic, but again, test `house1` and `house2` parameters and return one of *them*.  

You don't know if people will always call your function with kinningParkHouse and parkAvenueHouse - indeed that's why we have parameters!